<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filterSettings</class>
 <widget class="QDialog" name="filterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Directory Patterns</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="dirIncActive">
              <property name="text">
               <string>Include directory names mask</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="dirIncludes">
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="dirExclActive">
              <property name="text">
               <string>...but excluding names mask:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="dirExcludes">
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>File Patterns</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="fileInclActive">
              <property name="text">
               <string>Include file names mask:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="fileIncludes">
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="fileExclActive">
              <property name="text">
               <string>...but excluding names mask:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="fileExludes">
              <property name="lineWrapMode">
               <enum>QPlainTextEdit::NoWrap</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>filterSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>filterSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirIncActive</sender>
   <signal>stateChanged(int)</signal>
   <receiver>filterSettings</receiver>
   <slot>checkChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirExclActive</sender>
   <signal>stateChanged(int)</signal>
   <receiver>filterSettings</receiver>
   <slot>checkChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileInclActive</sender>
   <signal>stateChanged(int)</signal>
   <receiver>filterSettings</receiver>
   <slot>checkChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileExclActive</sender>
   <signal>stateChanged(int)</signal>
   <receiver>filterSettings</receiver>
   <slot>checkChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkChanged(int)</slot>
 </slots>
</ui>
